charts/ingress-nginx/values.yaml

```diff
@@ -577,7 +577,9 @@ controller:
       # -- Annotations to be added to the internal controller service. Mandatory for the internal controller service to be created. Varies with the cloud service.
       # Ref: https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer
       annotations: {}
-      # -- Type of the internal controller service.
+      # -- Additional custom labeles specifically for the internal controller service.
+      labels: {}
+      # -- Type of the internal controller service
```

Comment:
I think you accidentally reordered the comments here.

Reply:
Yes, corrected.